Makefile

```diff
@@ -0,0 +1,91 @@
+
+VERSION := 0.1.0
```

Comment:
Should this match the version that the repository has elsewhere (i.e. `0.1.2` currently)?

Reply:
Ah, I see you updated to 0.1.1 in a later commit, making me think it is meant to align--my bad for not having tagged the 0.1.2.

Reply:
Yes. I don't know how to automate it more unless updating the value manually for the release